rsync2/PKGBUILD

```diff
@@ -0,0 +1,63 @@
+# Maintainer: Joseph Benden <joe@benden.us>
+
+pkgname=rsync2
+pkgver=3.1.3dev_msys2.7.0_r2
+pkgrel=0
+pkgdesc="A file transfer program to keep remote files in sync"
+arch=('i686' 'x86_64')
+url="https://github.com/jbenden/rsync"
+groups=('net-utils')
+license=('GPL3')
+depends=('libiconv' 'msys2-w32api-runtime')
```

Comment:
Remove msys2-w32api-runtime from depends. This package don't contain any exe/dll